code/drasil-printers/package.yaml

```diff
@@ -14,6 +14,7 @@ dependencies:
 - lens
 - pretty
 - containers
+- MissingH
```

Comment:
Out of curiosity, where is this used?

Reply:
It is used [here](https://github.com/JacquesCarette/Drasil/pull/3823/files?diff=unified&w=0#diff-1247766c67d1c8c19d75aacc0f2153498b823b7055a0aac385c064ae5f66cdb5R155). For the `replace` function imported from `Data.List.Utils`.

Reply:
Ah, gotcha!